server/routes/index.js

```diff
@@ -24,8 +21,6 @@ db.connect();
 
 const store = db.initSessionStore();
 
-//algolia.initSyncWithAlgolia();
```

Comment:
This line shouldn't be commented, it's our function to sanitize the data between our mongodb and algolia index

Reply:
Doing it